README.md

```diff
@@ -17,13 +17,16 @@ The Reaction documentation source is located in the [reaction-docs](https://gith
 ## Installation
 > Reaction requires [Meteor](https://www.meteor.com/install), [Node](http://nodejs.org/), [npm](https://www.npmjs.com/) and [ImageMagick](http://www.imagemagick.org/script/binary-releases.php)
 
+Developers using **Windows** should review the [Windows specific installation requirements for Meteor and Reaction](https://docs.reactioncommerce.com/reaction-docs/development/requirements).
+
 ```bash
+curl https://install.meteor.com | /bin/sh # installs Meteor
 git clone https://github.com/reactioncommerce/reaction.git
 cd reaction
-./reaction install
+./reaction pull
```

Comment:
Should we keep the pull here? It's not really necessary - unless you really are developing..